### PR TITLE
add otakusplit v1.1 and update default kemap, support via

### PR DESCRIPTION
**Pull request by takashiski**
* minor change otaku_split v1 to v1.1
* update default keymap v1
* add via keymap v1 and v1.1

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by takashiski** [comment]
Hi, I fix and update checklist.

**Comment by takashiski** [comment]
@drashna @fauxpark @zvecr @noroadsleft
Would you this pullrequest?

File: keyboards/otaku_split/ver1/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,                    KC_H,  KC_J,  KC_K,  KC_L,  KC_SCLN,  KC_QUOT,KC_NUHS,
+			KC_LSHIFT,KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,             KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT, KC_SLASH,KC_RO,KC_RSHIFT,
+			KC_LCTRL, KC_LGUI,KC_LALT,KC_MHEN,LT(CURSOR,KC_DEL),KC_SPC,KC_SPC,        LT(CURSOR,KC_SPC),LT(CURSOR,KC_ENT),LT(CURSOR,KC_BSPC),KC_HENK,LT(CURSOR,KC_KANA),KC_RALT,KC_APP,KC_RCTRL
+  ),
+  [CURSOR] = LAYOUT(
+			KC_GRAVE, KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,             KC_F6, KC_F7,  KC_F8,  KC_F9,  KC_F10,  KC_F11,  KC_F12,  KC_JYEN,KC_DEL,
+			KC_TAB,   KC_Q,  KC_UP,  KC_E,  KC_R,  KC_T,  KC_Y,      KC_Y,  KC_U,  KC_I,  KC_O,  KC_P,  KC_LBRC,KC_RBRC,KC_PSCR,
+			KC_ESC,  KC_LEFT,  KC_DOWN,  KC_RIGHT,  KC_F,  KC_G,                    KC_LEFT,  KC_DOWN,  KC_UP,  KC_RIGHT,  KC_SCLN,  KC_QUOT,KC_NUHS,
+			KC_LSHIFT,KC_HOME,  KC_END,  KC_PGUP,  KC_PGDN,  KC_INSERT,             KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT, KC_END,KC_UP,KC_HOME,
+			KC_LCTRL, KC_LGUI,KC_LALT,KC_MHEN,KC_TRNS,KC_SPC,KC_SPC,        KC_TRNS,KC_TRNS,KC_TRNS,KC_HENK,KC_TRNS,KC_LEFT,KC_DOWN,KC_RIGHT
```
Comment (drashna) [inline keyboards/otaku_split/ver1/keymaps/via/keymap.c:37]:
```suggestion
    BASE,
    CURSOR
};

const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  [BASE] = LAYOUT(
      KC_ESC, KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,             KC_6, KC_7,  KC_8,  KC_9,  KC_0,  KC_MINUS  ,KC_EQL,KC_JYEN,KC_BSPC,
      KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,      KC_Y,  KC_U,  KC_I,  KC_O,  KC_P,  KC_LBRC,KC_RBRC,KC_ENT,
      KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,                    KC_H,  KC_J,  KC_K,  KC_L,  KC_SCLN,  KC_QUOT,KC_NUHS,
      KC_LSHIFT,KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,             KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT, KC_SLASH,KC_RO,KC_RSHIFT,
      KC_LCTRL, KC_LGUI,KC_LALT,KC_MHEN,LT(CURSOR,KC_DEL),KC_SPC,KC_SPC,        LT(CURSOR,KC_SPC),LT(CURSOR,KC_ENT),LT(CURSOR,KC_BSPC),KC_HENK,LT(CURSOR,KC_KANA),KC_RALT,KC_APP,KC_RCTRL
  ),
  [CURSOR] = LAYOUT(
      KC_GRAVE, KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,             KC_F6, KC_F7,  KC_F8,  KC_F9,  KC_F10,  KC_F11,  KC_F12,  KC_JYEN,KC_DEL,
      KC_TAB,   KC_Q,  KC_UP,  KC_E,  KC_R,  KC_T,  KC_Y,      KC_Y,  KC_U,  KC_I,  KC_O,  KC_P,  KC_LBRC,KC_RBRC,KC_PSCR,
      KC_ESC,  KC_LEFT,  KC_DOWN,  KC_RIGHT,  KC_F,  KC_G,                    KC_LEFT,  KC_DOWN,  KC_UP,  KC_RIGHT,  KC_SCLN,  KC_QUOT,KC_NUHS,
      KC_LSHIFT,KC_HOME,  KC_END,  KC_PGUP,  KC_PGDN,  KC_INSERT,             KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT, KC_END,KC_UP,KC_HOME,
      KC_LCTRL, KC_LGUI,KC_LALT,KC_MHEN,KC_TRNS,KC_SPC,KC_SPC,        KC_TRNS,KC_TRNS,KC_TRNS,KC_HENK,KC_TRNS,KC_LEFT,KC_DOWN,KC_RIGHT
```

File: keyboards/otaku_split/ver1p1/config.h [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+//#define MAGIC_KEY_LAYER8         8
+//#define MAGIC_KEY_LAYER9         9
+//#define MAGIC_KEY_BOOTLOADER     B
+//#define MAGIC_KEY_BOOTLOADER_ALT ESC
+//#define MAGIC_KEY_LOCK           CAPS
+//#define MAGIC_KEY_EEPROM         E
+//#define MAGIC_KEY_EEPROM_CLEAR   BSPACE
+//#define MAGIC_KEY_NKRO           N
+//#define MAGIC_KEY_SLEEP_LED      Z
+
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/config.h:183]:
```suggestion
```

File: keyboards/otaku_split/ver1p1/config.h [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+//#define NO_ACTION_MACRO
+//#define NO_ACTION_FUNCTION
+
+
+/* Bootmagic Lite key configuration */
+// #define BOOTMAGIC_LITE_ROW 0
+// #define BOOTMAGIC_LITE_COLUMN 0
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/config.h:205]:
```suggestion
```

File: keyboards/otaku_split/ver1p1/readme.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+![otaku_split_bottom](https://booth.pximg.net/c/620x620/4394ec37-d0ff-4c92-8f78-5c08d0566da6/i/1398595/cc9eb113-46a9-42fa-aaed-c4007efbe45e_base_resized.jpg)
+
+
+This is JP layout based split keyboards.
+
+
+
+Keyboard Maintainer: [takashiski](https://github.com/takashiski)  
+Hardware Supported: promicro(atmega32u4) on Otaku Split rev.1 PCB  
+Hardware Availability: https://skyhigh-works.booth.pm/items/1398595
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/readme.md:13]:
```suggestion
* Keyboard Maintainer: [takashiski](https://github.com/takashiski)
* Hardware Supported: promicro(atmega32u4) on Otaku Split rev. 1.1 PCB
* Hardware Availability: https://skyhigh-works.booth.pm/items/1398595
```

File: keyboards/otaku_split/ver1p1/readme.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make otaku_split/rev1:default
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/readme.md:17]:
```suggestion
    make otaku_split/rev1p1:default
```

File: keyboards/otaku_split/ver1p1/rules.mk [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/rules.mk:11]:
```suggestion
```

File: keyboards/otaku_split/ver1p1/rules.mk [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/rules.mk:26]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
```

File: keyboards/otaku_split/ver1p1/rules.mk [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/rules.mk:29]:
```suggestion
```

File: keyboards/otaku_split/ver1p1/rules.mk [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/rules.mk:31]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/otaku_split/ver1p1/ver1p1.c [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+  return process_record_user(keycode, record);
+}
+
+void led_set_kb(uint8_t usb_led) {
+  // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+  led_set_user(usb_led);
+}
+
+*/
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/ver1p1.c:51]:
```suggestion
```

**Comment by takashiski** [comment]
Thank you for telling tips! it is very useful!

File: keyboards/otaku_split/ver1p1/readme.md [hunk 11]
```diff
@@ -6,14 +6,12 @@
...
-Hardware Supported: promicro(atmega32u4) on Otaku Split rev.1 PCB  
-Hardware Availability: https://skyhigh-works.booth.pm/items/1398595
+* Keyboard Maintainer: [takashiski](https://github.com/takashiski)
+* Hardware Supported: promicro(atmega32u4) on Otaku Split rev. 1.1 PCB
+* Hardware Availability: https://skyhigh-works.booth.pm/items/1398595
 
 Make example for this keyboard (after setting up your build environment):
 
-    make otaku_split/rev1:default
+    make otaku_split/rev1p1:default
```
Comment (noroadsleft) [inline keyboards/otaku_split/ver1p1/readme.md:15]:
```suggestion
    make otaku_split/ver1p1:default
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

File: keyboards/otaku_split/ver1/keymaps/via/rule.mk [hunk 12]
```diff
@@ -0,0 +1,2 @@
+
+VIA_ENABLE=yes
```
Comment (fauxpark) [inline keyboards/otaku_split/ver1/keymaps/via/rule.mk:2]:
```suggestion
VIA_ENABLE = yes
```
This file must be named `rules.mk`.

File: keyboards/otaku_split/ver1p1/keymaps/via/rule.mk [hunk 13]
```diff
@@ -0,0 +1,2 @@
+
+VIA_ENABLE=yes
```
Comment (fauxpark) [inline keyboards/otaku_split/ver1p1/keymaps/via/rule.mk:2]:
```suggestion
VIA_ENABLE = yes
```
As above.

File: keyboards/otaku_split/ver1p1/rules.mk [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+SPLIT_KEYBOARD=yes
```
Comment (fauxpark) [inline keyboards/otaku_split/ver1p1/rules.mk:23]:
```suggestion
SPLIT_KEYBOARD = yes
```

File: keyboards/otaku_split/ver1/keymaps/via/keymap.c [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+      KC_LSHIFT,KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,             KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT, KC_SLASH,KC_RO,KC_RSHIFT,
+      KC_LCTRL, KC_LGUI,KC_LALT,KC_MHEN,LT(CURSOR,KC_DEL),KC_SPC,KC_SPC,        LT(CURSOR,KC_SPC),LT(CURSOR,KC_ENT),LT(CURSOR,KC_BSPC),KC_HENK,LT(CURSOR,KC_KANA),KC_RALT,KC_APP,KC_RCTRL
+  ),
+  [CURSOR] = LAYOUT(
+      KC_GRAVE, KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,             KC_F6, KC_F7,  KC_F8,  KC_F9,  KC_F10,  KC_F11,  KC_F12,  KC_JYEN,KC_DEL,
+      KC_TAB,   KC_Q,  KC_UP,  KC_E,  KC_R,  KC_T,  KC_Y,      KC_Y,  KC_U,  KC_I,  KC_O,  KC_P,  KC_LBRC,KC_RBRC,KC_PSCR,
+      KC_ESC,  KC_LEFT,  KC_DOWN,  KC_RIGHT,  KC_F,  KC_G,                    KC_LEFT,  KC_DOWN,  KC_UP,  KC_RIGHT,  KC_SCLN,  KC_QUOT,KC_NUHS,
+      KC_LSHIFT,KC_HOME,  KC_END,  KC_PGUP,  KC_PGDN,  KC_INSERT,             KC_B,  KC_N,  KC_M,  KC_COMM, KC_DOT, KC_END,KC_UP,KC_HOME,
+      KC_LCTRL, KC_LGUI,KC_LALT,KC_MHEN,KC_TRNS,KC_SPC,KC_SPC,        KC_TRNS,KC_TRNS,KC_TRNS,KC_HENK,KC_TRNS,KC_LEFT,KC_DOWN,KC_RIGHT
+  )
```
Comment (fauxpark) [inline keyboards/otaku_split/ver1/keymaps/via/keymap.c:38]: Pretty sure you need two extra empty layers here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**